src/txdb.h

```diff
@@ -65,7 +140,7 @@ class CCoinsViewDB : public CCoinsView
 class CBlockTreeDB : public CLevelDBWrapper
 {
 public:
-    CBlockTreeDB(size_t nCacheSize, bool fMemory = false, bool fWipe = false);
+    CBlockTreeDB(size_t nCacheSize, bool fMemory = false, bool fWipe = false, bool compression = true, int maxOpenFiles = 1000);
```

Comment:
We should set the default to compression=false and maxOpenFiles=64

Reply:
Fixed.